nixos/modules/programs/gui/vscode/default.nix

```diff
@@ -1,76 +1,68 @@
-{ pkgs, ... }: {
+{pkgs, ...}: {
   programs.vscode = {
     enable = true;
     package = pkgs.vscode;
-    enableUpdateCheck = false;
```

Comment:
Q: If I follow this correctly, we would not allow VS Code to check for extension updates. And instead we'd perform a rebuild via Nix OS?

What's your overall opinion on letting NixOS manage our say VS Code, instead of using something like [nix-ld](https://github.com/nix-community/nix-ld), which I just learned about?

I'm aware of letting Nix OS do it means its 100% reproducible etc, but I'm asking if you think it's worth the non-native approach that say VS Code offers with its extensions etc.